static/index.html

```diff
@@ -12,8 +12,10 @@
         <script src="ext/codemirror/codemirror.js"></script>
         <script src="asm-mode.js"></script>
         <script src="ext/jquery/jquery-2.2.2.min.js"></script>
+        <script src="ext/jquery-ui-1.11.4/jquery-ui.min.js"></script>
         <script src="ext/rison.js"></script>
         <script src="ext/clipboard.min.js"></script>
+        <script src="JSON-js/cycle.js"></script> <!-- for debug purpose only -->
```

Comment:
I was wondering what the external dependency on this new library was... can it be removed (along with the git submodule) (when the debugging is over!) please.